scm/driver/stash/webhook.go

```diff
@@ -100,8 +100,24 @@ func (s *webhookService) parsePullRequest(data []byte) (scm.Webhook, error) {
 	switch src.EventKey {
 	case "pr:opened":
 		dst.Action = scm.ActionOpen
+	case "pr:from_ref_updated":
+		// The request includes field "previousFromHash", which could be compared
+		// to the FromRef.Latestcommit to ensure there is actually a change,
+		// but there is unlikely need for that.
```

Comment:
Should these comments be added to the git commit instead? They would be available via e.g. blame feature in Github.